po/CMakeLists.txt

```diff
@@ -1,26 +1,30 @@
-set (ENABLE_NLS ON PARENT_SCOPE)
+cmake_minimum_required(VERSION 3.11)
+cmake_policy(VERSION 3.11)
 
-include (Gettext)
+set(ENABLE_NLS ON PARENT_SCOPE)
 
-if (XGETTEXT_FOUND)
+include(Gettext)
 
-  set (potfile "${PROJECT_PACKAGE}.pot")
+if (XGETTEXT_FOUND)
 
-  gettext_create_pot ("${potfile}"
-    PACKAGE ${PROJECT_PACKAGE}
-    VERSION ${PROJECT_VERSION}
-    OPTIONS --language=C++ -s --from-code=UTF-8
-            --keyword=_ --keyword=_F --keyword=N_
-            --keyword=C_:1c,2 --keyword=C_F:1c,2 --keyword=NC_:1c,2
+  set(potfile "${PROJECT_PACKAGE}.pot")
 
-    SRCFILES
-      "${PROJECT_SOURCE_DIR}/src/*.cpp"
+  gettext_create_pot("${potfile}"
+          PACKAGE ${PROJECT_PACKAGE}
+          VERSION ${PROJECT_VERSION}
+          OPTIONS --language=C++ -s --from-code=UTF-8
+          --keyword=_ --keyword=_F --keyword=N_
+          --keyword=C_:1c,2 --keyword=C_F:1c,2 --keyword=NC_:1c,2
 
-    GLADEFILES
-      "${PROJECT_SOURCE_DIR}/ui/*.glade"
-  )
+          SRCFILES
+          "${PROJECT_SOURCE_DIR}/src/*.cpp"
 
-  gettext_create_translations ("${potfile}" ALL NOUPDATE)
+          GLADEFILES
+          "${PROJECT_SOURCE_DIR}/ui/*.glade"
+          )
+  message(STATUS "SRCFILES: ${SRCFILES}")
+  message(STATUS "potfile: ${potfile}")
```

Comment:
It fails before these lines. They won't give you more information. Better to paste the content of Gettext here and annotate it in place.

Reply:
Do you mean in Gettext.cmake?
Now it works, but the gettext error is still there.

Reply:
the variable contains `../src/core/gui/toolbarMenubar/FontButton.cpp` so this happens in the call to xgettext.

Reply:
I think this is either a xgettext, cmake, or ninja bug. It also happens in all our pipeline runs.

Reply:
Yeah the bug seems to be related to either the CMake version or the change to PROJECT_NAME. I think it is the latter. For this PR it is okay to leave it as it is, since it does not break the full pipeline. We should definitely follow up on this though.

Reply:
There are 2 bugs, one, which is fixed by replacing PROJECT_PACKAGE with PROJECT_NAME.
And another one, which was there before (not introduced by this PR). The latter is mostly a dependency issue.

Reply:
Then this is resolved for now. Could you create a ticket for the latter issue?

Reply:
https://github.com/xournalpp/xournalpp/issues/3305